.github/workflows/ci.yml

```diff
@@ -5,6 +5,8 @@ on: [push, pull_request]
 jobs:
   container:
     runs-on: ubuntu-latest
+    env:
```

Comment:
@SkNuwanTissera You are pointing to dev to pull the latest connectors from dev-central.
But here https://github.com/ballerina-platform/asyncapi-triggers/blob/9ea41d70590439d992146d2d480da1f475c07bd4/.github/workflows/ci.yml#L17, 
the BCentral token is added for prod.

If we add prod token, this wont work for other environments.
Adding the token is necessary because without the token also we can pull connectors.

So can u remove that line where provide the central token?

Reply:
Thanks @LakshanSS Lets try it